README.md

```diff
@@ -7,13 +7,15 @@ I'm thinking of renaming the project to prevent confusion with the already exist
 
 Minecraft Version
 ------------------
-VoxelSniper was built against the Bukkit 1.15.2 API.
+VoxelSniper was built against the Bukkit 1.18.1 API.
```

Comment:
Why not 1.18.2, now that I think of it?

Reply:
Because I lifted it before 1.18.2 was out. No point in updating if there is no need. I think it improves backwards compatibility